### DISCUSSION TITLE
How many Lan cards from different companies be installed

### DISCUSSION THREAD

**Discussion by hi10bha** [Ysurac/openmptcprouter#1959] [Q&A]
Hi 
I want to know how many Lan cards can I install onboard for example I have brands like intel, ibm, rtf, uplink, dlink etc, so how can I embed this lan cards in my system, I know I may sound a little foolish but I am a complete new to this. thanks help much appreciated in advance

**Comment by Ysurac** [comment]
As many as you want.
But for WAN support, on x86_64 up to 16 connections are supported on latest release.

**Comment by hi10bha** [comment]
So is there any particular models af lab cards allowed or any brand , and
what is the structure to follow for installing the drivers

For example
TP-Link TG-3468 10/100/1000Mbps Gigabit PCI Express Network Adapter

And let’s say I want to use it to get signals in the system to use as
internet source


On Thu, 15 Jul 2021 at 22:50, Ycarus (Yannick Chabanois) <
***@***.***> wrote:

> As many as you want.
> But for WAN support, on x86_64 up to 16 connections are supported on
> latest release.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ysurac/openmptcprouter/discussions/1959#discussioncomment-1009702>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS6CQ67JAKXA4WFK7YWPEY3TX4KGNANCNFSM5AN47P2Q>
> .
>
-- 
Everybody is born Right handed,
Only gifted overcome it.

**Reply by Ysurac** [reply]
All cards supported by OpenWrt are supported, so there is many cards supported.
Drivers can be installed if need by System->Software

**Comment by hi10bha** [comment]
Thanks a lot for your email
Appreciate your help, if u can just throw some light on  system> software ,
how, m really sorry but totally new to this all,

Thanks in advance.


On Thu, 15 Jul 2021 at 23:08, Ycarus (Yannick Chabanois) <
***@***.***> wrote:

> All cards supported by OpenWrt are supported, so there is many cards
> supported.
> Drivers can be installed if need by System->Software
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ysurac/openmptcprouter/discussions/1959#discussioncomment-1009775>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AS6CQ6Z54CKH5MBO6K2DIJDTX4MJLANCNFSM5AN47P2Q>
> .
>
-- 
Everybody is born Right handed,
Only gifted overcome it.
